starknet-foundry/crates/cast/src/starknet_commands/account/deploy.rs

```diff
@@ -0,0 +1,18 @@
+use camino::Utf8PathBuf;
+use clap::Args;
+use starknet::core::types::FieldElement;
+
+#[derive(Args, Debug)]
+#[command(about = "Deploy an account to the Starknet")]
+pub struct Deploy {
+    /// Path to the file where the account secrets are stored
```

Comment:
what is this file? is it something different than accounts-file ?

Reply:
sooo, which one is it? :D

Reply:
This is accounts.json 😅

Reply:
You mean `~/.starknet_accounts/starknet_open_zeppelin_accounts.json` (by default) ? than why not use the --accounts-file flag?

Reply:
Right, I've already changed it in this next PR 😅